### PR TITLE
pkg/process/{model,config}: force using zstd v0.5 to avoid breaking with the intake and fix agent bin path on windows

### PR DESCRIPTION
**Pull request by sfluor**
### What does this PR do?

There was some issues decoding process/containers payloads encoded with zstd `1.3.0` this will force using zstd `0.5` to avoid any issues.

It also fixes the following error on windows:

image

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by kevinconaway** [comment]
Unfortunately backwards compatibility wasn't added to zstd until 0.8.1, I believe

**Comment by kevinconaway** [comment]
Where else is zstd being used ?

**Review by kevinconaway — COMMENTED** [review]
It might be good to have a backwards compatibility test here.  Perhaps a binary blob compressed with 0.5 that we can reliably uncompress

**Comment by sfluor** [comment]
It's the only place where it's used for processes / network, I will add a test 👍

File: Gopkg.toml [hunk 1]
```diff
@@ -10,6 +10,11 @@
   name = "github.com/DataDog/zstd"
   version = "~v1.3.0"
 
+[[constraint]]
+  name = "github.com/DataDog/zstd.v0.5"
+  source = "github.com/DataDog/zstd"
+  revision = "2b373cbe6ac0c8e6960bbd18026ceb269eef89f5"
```
Comment (sunhay) [inline Gopkg.toml]: Is this revision the same one we used previously in the process-agent?

Comment (sfluor) [reply Gopkg.toml]: it is 👍

File: pkg/process/config/config_windows.go [hunk 2]
```diff
@@ -13,7 +13,7 @@ var (
 	defaultLogFilePath = "c:\\programdata\\datadog\\logs\\process-agent.log"
 
 	// Agent 6
-	defaultDDAgentBin = "c:\\Program Files\\Datadog\\Datadog Agent\\embedded\\agent.exe"
+	defaultDDAgentBin = "c:\\Program Files\\Datadog\\Datadog Agent\\bin\\agent.exe"
```
Comment (sunhay) [inline pkg/process/config/config_windows.go]: When did this location change? Is there a PR? I wonder if it broke a previous build?

File: Gopkg.toml (hunk 1 above)
Comment (sfluor) [reply Gopkg.toml]: https://github.com/DataDog/datadog-process-agent/blob/master/Gopkg.lock#L99-L104

File: pkg/process/config/config_windows.go (hunk 2 above)
Comment (sfluor) [reply pkg/process/config/config_windows.go]: @derekwbrown might know better on this

File: pkg/process/model/message_test.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestDecodeZstd05Payload ensures backward compatibility with our intake
+func TestDecodeZstd05Payload(t *testing.T) {
+	file := "./test_zstd.0.5.dump"
```
Comment (kevinconaway) [inline pkg/process/model/message_test.go]: There is a convention for this, [`testdata`](https://dave.cheney.net/2016/05/10/test-fixtures-in-go)

**Review by kevinconaway — DISMISSED** [review]
small nit about the location of the test data but otherwise +1

File: pkg/process/model/message_test.go (hunk 3 above)
Comment (sfluor) [reply pkg/process/model/message_test.go]: 👍

**Review by kevinconaway — APPROVED** [review]

**Pull request merged**